acasclient/ddict.py

```diff
@@ -36,18 +40,14 @@ class ACASDDict(DDict):
     def __init__(self, code_type, code_kind):
         super(ACASDDict, self).__init__(code_type, code_kind, self.CODE_ORIGIN)
 
-    def get_values(self, client):
+    def update_valid_values(self, client):
+        """Get the valid values for the DDict."""
         valid_codetables = client.get_ddict_values_by_type_and_kind(
             self.code_type, self.code_kind)
         self.valid_values = [val_dict['code'] for val_dict in valid_codetables]
-        if self.valid_values == []:
+        if not self.valid_values:
```

Comment:
Just noting that @iamanandkr comment about using `self.valid_values is None` applies here as well

Reply:
Right, but here we are actually setting `self.valid_values` on the line above, so I think it will always be either empty list or populated list.